qdyn/output.f90

```diff
@@ -497,8 +519,13 @@ subroutine ox_init(pb)
   pb%ox%fmt(1) = "(i15)"
   ! Time needs higher precision
   pb%ox%fmt(2) = "(e24.14)"
+  ! Theta may require even higher precision
+  pb%ox%fmt(7) = "(e26.16)"
```

Comment:
In your opinion, is it necessary to have these excessively high precisions? Eyup suggested that a higher output precision will give more accurate restarts, but at the same time it will increase the output size on disk, and time to read/process. I increased the default precision to `(e20.12)` in `release/3.0.0`, which is already quite a step up from the previous one.

Reply:
Back then I used Eyup's suggestion, but I haven't tred restarting a model with different precisions and comparing the differences to comment further on this. You sugggest lower the precision of theta also to (e20.12)?

Reply:
If we stick to `(e20.12)` by default, then theta will automatically follow this. I'm not sure increasing the precision further will make much difference, since `real` only has 15-17 significant digits. The best solution (by far) would be to store the data in HDF5 or something, but that's another story...

Reply:
ok I'll change the precision back to (e20.12)